### PR TITLE
test: e2e contract test mint

### PR DESCRIPTION
**Pull request by matt-user**
closes FRO-596

### CODE REVIEW

File: packages/e2e-contract-tests/.eslintrc.cjs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+module.exports = {
```
Comment (luizstacio) [inline packages/e2e-contract-tests/.eslintrc.cjs:1]: I think we don't need another lint config on the project as it's going to use the root one.

File: packages/e2e-contract-tests/.gitignore [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Logs
```
Comment (luizstacio) [inline packages/e2e-contract-tests/.gitignore:1]: Also we have a root ignore file on this one we should add only the project specific files

File: packages/e2e-contract-tests/package.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  "scripts": {
+    "dev:e2e-contracts": "vite",
+    "build:web": "vite build",
+    "build:contracts": "pnpm exec fuels typegen -i ./contracts/custom_asset/out/debug/*-abi.json -o ./src/contracts",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "@fuel-ui/css": "0.21.0",
+    "@fuel-ui/react": "0.21.0",
```
Comment (luizstacio) [inline packages/e2e-contract-tests/package.json:14]: To keep it at minimum deps we could use just react with default HTML styles

File: packages/e2e-contract-tests/playwright/e2e/MintAsset.test.ts [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    const walletPage = await getWalletPage(context);
+    // short address function copied from app package
+    await hasText(walletPage, shortAddress(assetId));
+
+    // test mint amount is correct
+    await hasText(walletPage, mintAmount);
+
+    // test gas fee is shown and correct
+    await hasText(walletPage, 'Fee (network)');
+    // 0.000000001 since we hardcode a gas price of 1 in our mint function
```
Comment (luizstacio) [inline packages/e2e-contract-tests/playwright/e2e/MintAsset.test.ts:56]: I didn't understand this part as the gasFee should be set on the chain level.

File: packages/e2e-contract-tests/src/vite-env.d.ts [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+declare module '*?script&module' {
+  const src: string;
+  export default src;
+}
+
+// Add script  importing (CommonJS)
+declare module '*?script' {
+  const src: string;
+  export default src;
+}
```
Comment (luizstacio) [inline packages/e2e-contract-tests/src/vite-env.d.ts:21]: Do we need all this envs definitions here?

File: packages/test-utils/package.json [hunk 6]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@fuel-wallet/test-utils",
```
Comment (luizstacio) [inline packages/test-utils/package.json:2]: We can add this as private for now, and release once we achieve a stable version

File: packages/test-utils/src/test-utils/fuelWallet.ts [hunk 7]
```diff
@@ -0,0 +1,88 @@
+import { type BrowserContext, type Page } from '@playwright/test';
```
Comment (luizstacio) [inline packages/test-utils/src/test-utils/fuelWallet.ts:1]: Nice work here 🤘

**Review by luizstacio — CHANGES_REQUESTED** [review]

File: packages/e2e-contract-tests/playwright/e2e/MintAsset.test.ts (hunk 4 above)
Comment (matt-user) [reply packages/e2e-contract-tests/playwright/e2e/MintAsset.test.ts:56]: yes gasFee is set on chain level, but we also use a gasPrice of 1 in the txParams of the mint function.  If the gasPrice is set higher then the network fee will also be higher right?

**Review by luizstacio — APPROVED** [review]

**Pull request merged**